src/sorcha/modules/PPCalculateApparentMagnitude.py

```diff
@@ -60,9 +61,9 @@ def PPCalculateApparentMagnitude(
         else:
             observations.rename(columns={"H_" + mainfilter: "H_filter"}, inplace=True)
 
-        verboselog("Calculating apparent magnitude in filter...")
-        observations = PPCalculateApparentMagnitudeInFilter(
-            observations, phasefunction, lightcurve=lightcurve, lightcurve_choice=lightcurve_choice
-        )
+    verboselog("Calculating apparent magnitude in filter...")
+    observations = PPCalculateApparentMagnitudeInFilter(
+        observations, phasefunction, lightcurve=lightcurve, lightcurve_choice=lightcurve_choice
```

Comment:
I am not sure if we should be applying a cometary phase curve or if this is already in the comet class. Let's get some configuration and a calculation of a comet mag at different distances from Colin before this pull request is evaluated

Reply:
@astronomerritt can follow-up on this

Reply:
I've emailed Colin.

Reply:
However: @fedorets I see what's been done here, but this wouldn't fix the issue by itself even if this is the way we want to do it. Note that PPCalculateSimpleCometaryMagnitude creates the TrailedSourceMag column by default. PPCalculateApparentMagnitudeInFilter operates on H_filter and also creates TrailedSourceMag, overwriting what was done by PPCalculateSimpleCometaryMagnitude. 

This could be easily fixed by changing the default colname in PPCalculateSimpleCometaryMagnitude.py to "H_filter" (line 8). Then you could remove the line 53 you've added here in PPCalculateCometaryMagnitude.